src/main/java/Iris/Iris.java

```diff
@@ -0,0 +1,34 @@
+package Iris;
+
+import java.util.Scanner;
+
+public class Iris {
+    private final Storage taskStorage;
+    private final Parser commandParser;
+    private static ToDoList toDoList;
```

Comment:
The class name "TodoList" might be confusing since todo is a subclass of task and the list is actually holding tasks. Maybe it would be better to rename it as TaskList